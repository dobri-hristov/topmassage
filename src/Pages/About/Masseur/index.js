import React from 'react'
import styles from '../About.module.css'

const Masseur = ({ imgPath, name, position, info }) => {
    return (
        <article className={styles.employee}>
            <img src={imgPath} alt={name} />
            <h3 className={styles.employeeName}>{name}</h3>
            <h5 className={styles.employeePosition}>{position}</h5>
            <p className={styles.employeeInfo}>{info}</p>
        </article>
    )
}

export default Masseur
