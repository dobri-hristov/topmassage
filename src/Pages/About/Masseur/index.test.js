import React from 'react'
import TestRenderer from 'react-test-renderer'
import Masseur from './index.js'

describe('About page/Masseur component', () => {
    test('should render correctly with data', () => {
        const component = TestRenderer.create(
            <Masseur
                imgPath="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/Mihaela_lvoe9a.jpg"
                name="Михаела Иванова"
                position="Мениджър"
                info="С магистърска степен специалност кинезитерапия, Михаела има дългогодишен опит както кинезитерапевт, консултант, 
                експерт и управител.Нейната сила и страст е в индивидуално диагностициране и прилагане на ефективни масажи
                за превенция и профилактика на Вашето здраве."
            />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})