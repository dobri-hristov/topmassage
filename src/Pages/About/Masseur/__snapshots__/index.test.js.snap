// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page/Masseur component should render correctly with data 1`] = `
<article
  className="employee"
>
  <img
    alt="Михаела Иванова"
    src="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/Mihaela_lvoe9a.jpg"
  />
  <h3
    className="employeeName"
  >
    Михаела Иванова
  </h3>
  <h5
    className="employeePosition"
  >
    Мениджър
  </h5>
  <p
    className="employeeInfo"
  >
    С магистърска степен специалност кинезитерапия, Михаела има дългогодишен опит както кинезитерапевт, консултант, 
 експерт и управител.Нейната сила и страст е в индивидуално диагностициране и прилагане на ефективни масажи
 за превенция и профилактика на Вашето здраве.
  </p>
</article>
`;
